File: demo/springboot/demo/src/main/java/com/demo/domain/dto/http/resonse/OssUserBucketFileItemListResponse.java
package com.demo.domain.dto.http.resonse;

import lombok.Data;

import java.util.List;

@Data
public class OssUserBucketFileItemListResponse {
    private String userId;
    private String bucketName;
    private List<OssUserBucketFileItemRow> items;
}

File: demo/kmp/shared/src/commonMain/kotlin/com/vectordemo/domain/dto/http/response/OssUserBucketFileItemListResponse.kt
package com.vectordemo.domain.dto.http.response

import kotlinx.serialization.Serializable

@Serializable
data class OssUserBucketFileItemListResponse(
    val userId: String? = null,
    val bucketName: String? = null,
    val items: List<OssUserBucketFileItemRow>? = null,
)
